packages/nc-gui-v2/components/smartsheet/sidebar/MenuTop.vue

```diff
@@ -186,8 +188,14 @@ function openDeleteDialog(view: Record<string, any>) {
       closeDialog()
 
       emits('deleted')
-      // return to the default view
-      activeView.value = views.value[0]
+      if (activeView.value === view) {
```

Comment:
shouldn't this be `activeView.value === view.value`?

Reply:
`view` is just a value passed from the template and not a reactive variable

Reply:
Ah okay, my bad then :)